pkg/network/usm/ebpf_gotls.go

```diff
@@ -208,6 +212,7 @@ func (p *goTLSProgram) GetStats() *protocols.ProtocolStats {
 
 // Stop terminates the uprobe attacher for GoTLS programs.
 func (p *goTLSProgram) Stop(*manager.Manager) {
+	p.procMon.Stop()
```

Comment:
why not encapsulate that inside the upobe attacher? call the stop inside attacher.Stop(), that way you don't need to store the pm reference in the ebpf program